### PR TITLE
Add release notes for 17.03.2-ee-6

### PR DESCRIPTION
**Pull request by joaofnfernandes**

### CODE REVIEW

**Comment by andrewhsu** [comment]
can we have separate PRs for this? we may need to split the release timeline of 17.03 and 1.13

**Comment by joaofnfernandes** [comment]
CS engine release notes now being tracked at #4363

**Comment by andrewhsu** [comment]
LGTM

(title of PR needs to be updated)

File: enterprise/17.03/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ it references. However, Docker EE also includes back-ported fixes
 defect fixes that you can use in environments where new features cannot be
 adopted as quickly for consistency and compatibility reasons.
 
+## Docker EE 17.03.2-ee-6 (2017-08-24)
```
Comment (mstanleyjones) [inline enterprise/17.03/release-notes.md]: The date isn't on the header line of the previous entries.

**Pull request merged**